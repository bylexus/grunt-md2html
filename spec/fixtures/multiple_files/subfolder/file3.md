File 3
=======

Base: <%= basepath %>
