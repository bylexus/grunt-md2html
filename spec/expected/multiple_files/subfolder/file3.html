<!DOCTYPE html>
<html>
    <head>
        <link rel="stylesheet" type="text/css" href="../../../spec/fixtures/style.css">
        <title></title>

    </head>

    <body>
        <h1 id="file-3">File 3</h1>
<p>Base: ../../../spec/fixtures</p>

    </body>
</html>
